image.yaml

```diff
@@ -105,14 +105,22 @@ modules:
           - name: jboss.container.eap.prometheus.config
           - name: os-eap-txnrecovery.run
             version: 'python3'
-          - name: os-eap-python
-            version: '3.6'
-           # At this point, all scripts must have been added to JBOSS_HOME and to custom galleon packages location
+
+          # At this point, all scripts must have been added to JBOSS_HOME and to custom galleon packages location
           - name: jboss.container.wildfly.galleon.build-feature-pack
           - name: jboss.container.wildfly.galleon.provision-server
           - name: jboss.container.eap.final-setup
-packages:
-      content_sets_file: content_sets.yml
+
+# note that building in OSBS using ODCS requires a content set to be specified, these are done in the overrides
+# cekit doesn't appear to allow this to be overriden from a default
+#packages:
+#      content_sets_file: content_sets.yml
+
+artifacts:
+  - name: maven-repo
+    target: maven-repo.zip
+    md5: 927cf0a61a4dccab4fb835d6a5fec2f0
```

Comment:
how can I check that the checksum correspond to the correct Maven repository for EAP 7.3CR2?

Reply:
The only way to do this is to compare what is here with the checksums created when the maven zip is built by prod. I'll link you the internal URL seperately.